### PR TITLE
registering jeringeorge98

### PR DESCRIPTION
**Pull request by jeringeorge98**
Hey maintainer!
Thanks for the work you do!!

Kindly accept my request

### CODE REVIEW

File: content/participants/ttjc@jeringeorge98/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+I  love developing web and mobile applications,familiar and well versed with frameworks like REACT NATIVE||ReactJs||ExpressJs||NodeJs.Love challenging tasks and problem solving ,if given an oppurtunity would love to build applications that solve real world problems. 
+
+## Why are you taking this challenge?
+
+I need a job as i will be graduating July 2020.
+
+I want to learn and be part of the dev community find mentors who could help me in my journey to becoming a great developer.
+
+I also want to know more about Open Source development and would like to conntribute in future as most of the things i have learnt has been with the help of Open source software.
```
Comment (khusharth) [inline content/participants/ttjc@jeringeorge98/index.md:24]: Hey! @jeringeorge98 i found a small typo here:
* conntribute

File: content/participants/ttjc@jeringeorge98/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## something interesting about you?
+If I am not coding ,Iam reading books on economics and finance,reading about dispruptive technologies like blockhain,love playing chess and discussing football.
```
Comment (khusharth) [inline content/participants/ttjc@jeringeorge98/index.md:27]: I am reading*

File: content/participants/ttjc@jeringeorge98/index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+_Github link_:https://github.com/jeringeorge98/Bug-tracking-software-with-vanilla-Js-
+
+#### Simple React-Native application and Authentication with firebase  
+
+_Description_:
+Mobile application built with React Native ,which demonstrates simple authentication ,Users are authetcated with firebase api.
+_Stack_:
+React-Native ,Firebase
+
+_Github link_:https://github.com/jeringeorge98/Bug-tracking-software-with-vanilla-Js-
```
Comment (khusharth) [inline content/participants/ttjc@jeringeorge98/index.md:75]: The github link here is same as your previous project. Kindly update this.

File: content/participants/ttjc@jeringeorge98/index.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+### Internship
+Worked at a startup for 7 months were I worked on a co-working mobile application.The application was built with REACT NATIVE ,I was involved throughout the process from production to development.
+
+_Roles ad Responsibilties_: Included buidling frontend User interfaces on react native ,leveraging 3 party APIS, and building RESTful APIS to connect to the backend.
```
Comment (khusharth) [inline content/participants/ttjc@jeringeorge98/index.md:80]: Some more typos:
* and
* building

Comment (jeringeorge98) [reply content/participants/ttjc@jeringeorge98/index.md:80]: Sorry for the typos i will correct them and send it again

Comment (jeringeorge98) [reply content/participants/ttjc@jeringeorge98/index.md]: Hey khusharth i have resolved the typos and changed the link please take alook and accept my request.

Thank You!

**Pull request merged**